auth/server/src/main/java/com/seagates3/policy/PolicyValidator.java

```diff
@@ -229,4 +236,226 @@ boolean isPrincipalIdValid(Principal principal) {
       validateActionAndResource(List<Action> actionList,
                                 List<Resource> resourceValues,
                                 String inputBucket);
+
+ protected
+  ServerResponse validatePolicyElements(
```

Comment:
does aws sdk provide validate policy method, we should reuse that method

Reply:
as confirmed by @shalakadharap, No. aws sdk dont provide such method.